plugins/flowdock/app/assets/javascripts/flowdock/application.js

```diff
@@ -1,112 +1,2 @@
 //= require jquery_mentions_input/jquery.elastic.source.js
```

Comment:
tried updating it ... it's a giant pain ... leaving it like this for now ... we can pull mentions into core if slack uses it too ...